src/main/groovy/com/google/protobuf/gradle/ExecutableLocator.groovy

```diff
@@ -36,6 +37,7 @@ import org.gradle.api.Named
  * configured, the plugin should try to run the executable from system search
  * path.
  */
+@CompileDynamic
```

Comment:
Some classes like this one, are able to be compiled statically. But this PR is just upgrading the compiler, I am fine with just annotating everything as dynamic compile.

Reply:
I noticed that as well. But swapping from dynamic to static also changes method resolution. I also decided it is probably better to leave things dynamic, at least for now.